modules/raft/src/main/java/org/apache/ignite/raft/jraft/util/Utils.java

```diff
@@ -123,6 +126,29 @@ public static Future<?> runClosureInThread(ExecutorService executor, final Closu
         return runClosureInThread(executor, done, Status.OK());
     }
 
+    private static final long pid = ProcessHandle.current().pid();
```

Comment:
This looks like a debug piece of code. Should it be removed?

Reply:
You're right, I completely forgot about this